### ISSUE TITLE
Jaccard loss

### ISSUE THREAD

**Issue by innat** [keras-team/keras#16431]
Another useful loss function for semantic segmentation tasks. It was available in [keras-contrib](https://github.com/keras-team/keras-contrib), [losses/jaccard.py](https://github.com/keras-team/keras-contrib/tree/master/keras_contrib/losses)

**Comment by innat** [comment]
@qlzh727 It would be great to have this loss function from core keras. But as it's mostly cv-related loss function, isn't it fitter to have it from keras_cv; if cv-related stuff moves to keras_cv from keras in the future?  Also, if we have [Dice loss](https://github.com/keras-team/keras/issues/16431#issue-1207415524) from keras_cv and Jaccard loss from keras, it might be a confusing UX, IM0.

**Comment by qlzh727** [comment]
I will check with API owner for this question, and see where we want to place this loss. Will update later this week.

**Comment by bhack** [comment]
Just for the record: https://github.com/keras-team/keras-contrib/pull/533
